static/js/components/GcnSelectionForm.jsx

```diff
@@ -206,7 +207,7 @@ const GcnSelectionForm = ({ gcnEvent, setSelectedLocalizationName }) => {
 
   if (
     !gcnEvent ||
-    // !gcnEventSources ||
+    !gcnEventSources ||
```

Comment:
Couldn't we still charge the gcn selection form as soon as we have the event and its localization ? And then the options to display or not display sources, observations, instruments, galaxies ... appear as those become available? Or they are already there the buttons are deactivated and they get clickable as it loads ?

Reply:
Just to follow the async behaviour we expect from the backend you know. And to not end up having a whole set of features deactivated when just one of them really is.

Reply:
@Theodlz I pushed a commit with your disable button trick... give it a try when you have a chance.